data/Minecraft_Pi_Reborn_Server

```diff
@@ -0,0 +1 @@
+https://jenkins.thebrokenrail.com/job/minecraft-pi-reborn/job/master/lastSuccessfulBuild/artifact/out/minecraft-pi-reborn-server-2.3.2-amd64.AppImage
```

Comment:
Also, this link will break when a new version is released, because it specifically refers to the last successful build.

Reply:
That's ok, because all that is needed is for the test to pass once. We do not keep track of each version.